### ISSUE TITLE
Qgis is not packaged with Grass Gis 7.4 app

### ISSUE THREAD

**Issue by Lvaskz** [flathub/org.qgis.qgis#7]
Hi, I noticed qgis has many features linked with Grass Gis 7.4, but qgis 3.2 has not,because is not installed. the features with grass are not working. In the installation in windows Grass Gis 7.4 come by default.

**Comment by Lvaskz** [comment]
Hi, I reinstall qgis and the issues remain. Same with saga gis

**Comment by Lvaskz** [comment]
![ksnip_20181013-234927](https://user-images.githubusercontent.com/29207169/46913069-53c55580-cf43-11e8-8105-d1bdbefbdcc2.png)

**Comment by Lvaskz** [comment]
there are not something to update in flathub

**Comment by Lvaskz** [comment]
Hi, can check this again? I am not getting any update from your improvements.

**Comment by aleixpol** [comment]
I've had positive feedback on this issue, but in any case, if you can investigate a bit that would be really helpful.

**Comment by aleixpol** [comment]
Right, I see what you mean. Now it's not finding the libraries... ugh.
